_includes/asg/ebs-template.md

```diff
@@ -10,21 +14,26 @@ Ensure that these EBS volumes are created in the same region as the auto scaling
 
 Record the EBS volume ID (e.g. _vol-04e2283f1925ec9ee_), this will be passed in to PX as a parameter as a storage device.
 
-**For PX version > 1.3**
+### Using a template specification
 
-A set of EBS volume templates can be specified in the following comma-separated format
+For PX 1.3 and higher, you can specify a template spec which will be used by Portworx to create new EBS volumes.
```

Comment:
Can we make it explicit in UI as well that this is supported only for 1.3

Reply:
Removing it from the dublin spec: https://github.com/portworx/px-installer/pull/79